NDTensors/src/linearalgebra/linearalgebra.jl

```diff
@@ -218,7 +218,7 @@ function LinearAlgebra.eigen(
   use_relative_cutoff::Bool = get(kwargs, :use_relative_cutoff, use_relative_cutoff)
 
   matrixT = matrix(T)
-  if any(!isfinite, matrixT)
+  if any(!isfinite, parent(matrixT))
```

Comment:
I'm guessing this was to avoid element-wise operations on GPU? If so, probably good to leave a comment about that since hopefully that won't be necessary with future designs of the storage type.